framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2651,7 +2651,7 @@ protected void parseStubFiles() {
      * @return the annotation mirror for anno
      */
     @Override
-    public AnnotationMirror getDeclAnnotation(Element elt,
+    public final AnnotationMirror getDeclAnnotation(Element elt,
```

Comment:
There is a `getDeclAnnotations(Element elt)` version which is public, but non-final.
Maybe we should make public methods which are not intended to be overridden final?
Let's just keep this as it is and see whether we need to change this later.